mojo/core/broker_castanets.cc

```diff
@@ -246,23 +243,18 @@ void BrokerCastanets::OnBufferSync(uint64_t guid_high, uint64_t guid_low,
     return;
   }
 
-  int fd = base::SharedMemoryTracker::GetInstance()->Find(guid);
-  base::subtle::PlatformSharedMemoryRegion handle;
-  if (fd < 0) {
     base::SharedMemoryCreateOptions options;
```

Comment:
Looks like indendation is not proper from L 246 to L257 can you cross check once.